governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+
+# [Executive Proposal] Enhanced Dai Savings Rate Activation, Spark Protocol Debt Ceiling Increase, Set Harbor Trade Debt Ceiling to Zero, AVC Members Rewards for Q2, DAO Resolution for Monetalis Clydesdale and Other Actions - August 2, 2023
+
+The Governance Facilitators, Dewiz, Sidestream and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The Enhanced Dai Savings Rate (EDSR) will be activated, increasing the DSR to 8%.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 200 million DAI. 
+- The Harbor Trade Credit vault's Debt Ceiling will be set to zero DAI, and the vault will be put into soft liquidation. 
+- AVC Members' Participation Rewards for Q2 will be distributed. 
+- A DAO Resolution pertaining to the Monetalis Clydesdale James Asset Trust 1 (JAT1) structure will be approved. 
+- A Spark Protocol Proxy Spell will be triggered to implement changes to the DAI interest rate strategy, WETH market parameters and DAI market parameters.
+- 2 million DAI of Launch Project funds will be distributed to a wallet controlled by the Accessibility Scope Facilitator for growth and marketing expenses.
+- The New Silver vault's RWA Agreement parameter will be updated. 
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with links showing its provenance. "Authorization" links to the official document(s) authorizing the proposed item's inclusion in the Executive Vote; this can be a governance poll, the Atlas and/or a Scope Alignment Artifact. "Reasoning" links to a MakerDAO Forum post providing the rationale and additional context for each item. Where needed, "Authorizing Parties" may also be listed; these may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Enhanced Dai Savings Rate (EDSR) Activation
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmcTRPLx)\
```

Comment:
I feel like it might be worth putting slightly more info here. Maybe:
```suggestion
*Authorization*: [Successful Governance Poll - 2023-07-24](https://vote.makerdao.com/polling/QmcTRPLx)\
```

Successful to make it more idiot-proof, maybe unneeded. Feels like the date is useful to distinguish between recent polls, and long-ago ones.